StripeConnect/StripeConnectTests/Internal/Webview/ConnectComponentWebViewControllerTests.swift

```diff
@@ -463,8 +463,9 @@ class ConnectComponentWebViewControllerTests: XCTestCase {
         let componentManager = componentManagerAssertingOnFetch()
         let session = try FinancialConnectionsSessionMock.default.make()
 
-        let financialConnectionsPresenter = MockFinancialConnectionsPresenter { apiClient, secret, connectedAccountId, vc in
-            XCTAssert(apiClient === componentManager.apiClient)
+        let financialConnectionsPresenter = MockFinancialConnectionsPresenter { compManager, secret, connectedAccountId, vc in
+            XCTAssert(compManager.apiClient == componentManager.apiClient)
+            XCTAssert(compManager.publicKeyOverride == componentManager.publicKeyOverride)
```

Comment:
Doublechecking – STPAPIClient subclasses NSObject and doesn't override `isEqual`, so I think this should just do pointer-comparison, right? Just want to make sure the equality comparison does what we think it should.

Non-blocking: Also, if we're using `==` anyway, this could be `XCTAssertEqual` which would provide a better failure message if the test fails.
```suggestion
            XCTAssertEqual(compManager.apiClient, componentManager.apiClient)
            XCTAssertEqual(compManager.publicKeyOverride, componentManager.publicKeyOverride)
```

Reply:
Yeah this should be doing pointer comparison.